pym/bob/scm/git.py

```diff
@@ -81,25 +92,57 @@ def getProperties(self):
                     ("refs/heads/" + self.__branch))
             )
         }]
+        for key, val in self.__remotes.items():
+            properties[0].update({GitScm.REMOTE_PREFIX+key : val})
+        return properties
 
     def asScript(self):
+        remotes_array = """
+# create an array of all remotes for this repository
+declare -A BOB_GIT_REMOTES=( [origin]="{URL}" )""".format(URL=quote(self.__url))
```

Comment:
The double-quotes in `[origin]="{URL}"` must be removed now. The `quote()` function will insert them as needed and this will collide if there are already some... (same below)

Reply:
done